operator/v1/0000_70_dns-operator_00.crd.yaml

```diff
@@ -44,6 +44,10 @@ spec:
                       pattern: ^(0|([0-9]+(\.[0-9]+)?(ns|us|µs|μs|ms|s|m|h))+)$
                       type: string
                   type: object
+                forceTCP:
```

Comment:
I added this as `spec.forceTCP` which added forceTCP to all upstream (forward) configurations, including the default one (/etc/resolv.conf). However, it could be argued that `spec.forceTCP` shouldn't be global and instead only for the default upstream, and we should also add `spec.upstreamResolvers.upstreams.forceTCP` to individually configure forward zones.

Reply:
Whoops, just read https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#do-not-use-boolean-fields

Fixing that.